File: RxDemo/src/main/java/com/tamsiree/rxdemo/interfaces/ShopCartInterface.java
package com.tamsiree.rxdemo.interfaces;

import android.view.View;

/**
 *
 * @author tamsiree
 * @date 16-11-13
 */
public interface ShopCartInterface {
    void add(View view, int postion);
    void remove(View view, int postion);
}

File: RxDemo/src/main/java/com/tamsiree/rxdemo/interfaces/ShopCartInterface.kt
package com.tamsiree.rxdemo.interfaces

import android.view.View

/**
 *
 * @author tamsiree
 * @date 16-11-13
 */
interface ShopCartInterface {
    fun add(view: View?, position: Int)
    fun remove(view: View?, position: Int)
}